front/lib/resources/trigger_resource.ts

```
@@ -239,6 +243,11 @@ export class TriggerResource extends BaseResource<TriggerModel> {
           auth,
           trigger: this,
         });
+      case "webhook":
+        // TODO: Implement webhook workflow when ready.
+        return new Err(
```

throw here (rather than returning err)